app/hotels/src/filter/hotelFacilities/HotelFacilitiesPopup.js

```diff
@@ -120,17 +121,29 @@ export default class HotelFacilitiesPopup extends React.Component<
     facilitiesListKeys.forEach((key, i) => {
       const facility = facilitiesList[key];
       checkboxes.push(
-        <Checkbox
-          key={i}
-          isChecked={facilities.indexOf(key) >= 0}
-          onPress={this.handleCheckboxOnPress(key)}
-          style={i < facilitiesListKeys.length - 1 && styles.delimiter}
-        >
-          <View style={styles.checkbox}>
-            <Icon name={facility.icon} size={20} style={styles.facilityIcon} />
-            {facility.text}
-          </View>
-        </Checkbox>,
+        <React.Fragment key={i}>
+          <Checkbox
+            isChecked={facilities.indexOf(key) >= 0}
+            onPress={this.handleCheckboxOnPress(key)}
+          >
+            <View style={styles.checkbox}>
+              <TextIcon
+                code={facility.icon}
+                orbit={true}
+                style={styles.facilityIcon}
+              />
+              {facility.text}
+            </View>
+          </Checkbox>
+          {i < facilitiesListKeys.length - 1 && (
+            <View style={styles.separatorEnd}>
+              <SeparatorTrimmed
+                gapSizeStart={24 + 7}
```

Comment:
Is it a reason why it is 24 + 7 rather than 31?

Reply:
I wrote it like this for readability, it's iconSize + margin, but I guess it wasn't that obvious 😆The margin is actually 10, but it wasn't properly aligned in the emulator, when I used orbit icons, so I changed it to 7 and now it doesn't make sense 😆I'll fix it.

Reply:
In that case it would have been better to make to variables, iconSize & margin and done `gapSizeStart={iconSize + margin}`